apps/web/src/components/templates/in-app-editor/ContentContainer.tsx

```diff
@@ -54,7 +36,7 @@ export function ContentContainer({
         dangerouslySetInnerHTML={{
           __html: content as string,
         }}
-        onKeyUp={(e: any) => handleChange(e)}
+        onKeyUp={(e: any) => onChange(e.target.innerHTML)}
```

Comment:
only the `innerHTML` value works well with `dangerouslySetInnerHTML` for multiline content